### PR TITLE
Physics Review

### PR DESCRIPTION
**Pull request by sinkingsugar**

### CODE REVIEW

File: manual/physics/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Rigid body colliders will move based on the forces (such as gravity and collisio
 
 Entities with colliders attached to them, should not be moved directly by their **Transform** property, but instead should be manipulated by applying forces.
 
-However if your entity is controlled by script or animation most of the time, but only sometimes needs to interact with other physics objects, you can set the **Is Kinematic** flag for a Rigid body collider. The collider will still raise collision events, but will not respond to them. When the collider needs to start interacting with other colliders again, simply disable the **Is Kinematic** flag in your script.
+However if your entity is controlled by script or animation most of the time, but only sometimes needs to interact with other physics objects, you can set the **Is Kinematic** flag for a Rigid body collider. The collider will still raise collision events, but will not respond to them. When the collider needs to start interacting with other colliders again, simply disable the **Is Kinematic** flag in your script. When **Is Kinematic** flag is set, you will be able to move the Entity and thus the rigid-body from the **Transform** itself.
```
Comment (sinkingsugar) [inline manual/physics/index.md]: You get the idea,  wording needs review

Comment (sinkingsugar) [inline manual/physics/index.md]: !fix

File: manual/physics/index.md [hunk 2]
```diff
@@ -8,16 +8,17 @@ This section will show you how to enable physics in Game Studio, how to use the
 
 ## Basics
 
-Physics simulation is the process of simulating how physical objects interact with each other, and how they react to forces such as gravity or collisions. In Xenko we refer to physics objects as **Colliders**. In Xenko we use 2 types of colliders:
+Physics simulation is the process of simulating how physical objects interact with each other, and how they react to forces such as gravity or collisions. In Xenko we refer to physics objects as **Physics Components**. In Xenko we have 3 types of physics components:
```
Comment (sinkingsugar) [inline manual/physics/index.md]: While I agree that Collider is simpler as word, it is very confusing code side, Colliders  don't exist in the code side, only Physics Componets derivatives exist now.
Also Collider can be confused with the collider shape itself.

If we use these words, we should change the code as well I believe.

Comment (sinkingsugar) [inline manual/physics/index.md]: !resolve

File: manual/physics/index.md (hunk 1 above)
Comment (sinkingsugar) [inline manual/physics/index.md]: !fixed

Comment (sinkingsugar) [inline manual/physics/index.md]: !resolve

File: manual/physics/index.md [hunk 3]
```diff
@@ -8,16 +8,17 @@ This section will show you how to enable physics in Game Studio, how to use the
...
 
-* [Rigid body colliders](rigid-body.md)
-* [Static colliders](static-collider.md)
+* [Rigid body](rigid-body.md)
+* [Static collider](static-collider.md)
+* [Character] (character-controller.md)
 
 Rigid body colliders will move based on the forces (such as gravity and collisions) applied to them. Static colliders will cause a collision when a rigid body collider collides with it, but will not move themselves. Typically static game elements such as walls and floors will be static colliders, and game elements (such as players, cars, etc.) will be rigid body colliders.
```
Comment (xenux) [inline manual/physics/index.md]: test

Comment (xenux) [inline manual/physics/index.md]: !fix

Comment (xenux) [inline manual/physics/index.md]: !resolved

File: manual/physics/index.md (hunk 1 above)
Comment (teo-tsirpanis) [inline manual/physics/index.md]: !fixed

Comment (teo-tsirpanis) [inline manual/physics/index.md]: !resolve

**Pull request closed without merging**